txscript/scriptnum.go

```diff
@@ -152,11 +156,6 @@ func (n scriptNum) Int32() int32 {
 // makeScriptNum interprets the passed serialized bytes as an encoded integer
 // and returns the result as a script number.
 //
-// Since the consensus rules dictate the serialized bytes interpreted as ints
```

Comment:
Rather than removing this important information, I'd prefer it simply be modified slightly to reflect the existence of a parameter.  Perhaps:

```Go
// Since the consensus rules dictate that serialized bytes interpreted as ints
// are only allowed to be in the range determined by the maximum number of
// bytes, an error will be returned when the provided bytes would result in a
// number outside of that range.  In particular, the range for the vast majority
// of opcodes dealing with numeric values are limited to 4 bytes and therefore
// will pass that value to this function resulting in an allowed range of
// [-2^31 + 1, 2^31 - 1].
```